performance.rst

```diff
@@ -217,6 +217,44 @@ deployment process too):
   used in your application and prevents Composer from scanning the file system for
   classes that are not found in the class map. (see: `Composer's autoloader optimization`_).
 
+Disable Dumping the Container as XML in Debug Mode
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In debug mode, Symfony compiles the :doc:`service container </service_container>`
+into an XML file. This XML file is used by various commands such as
+``debug:container`` and ``debug:autowiring``. You can disable dumping of the XML
```

Comment:
Perhaps write why one would want to disable it
Time saving at cache clear etc

Reply:
Made some changes, do you like this more?

Reply:
yes it is worth documenting the why, not only the how :)
so this new sentence is better imho 👍🏻